prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+    var $container = document.querySelector('#container');
+    console.log($container);
+
+    var $heading = document.createTextNode('Hello World');
```

Comment:
For this exercise we want to create a new `'h1` element and then set its `.textContent` to `'Hello World'`. Then, when the `h1` element is ready, we append the `h1`.

Reply:
![updated dom creation](https://user-images.githubusercontent.com/88108910/129614330-153704ad-ac40-4181-a468-f17e952ee8e2.png)